### PR TITLE
API get reviews in detail book document

### PR DESCRIPTION
**Pull request by mvn-caonguyen-dn**
## Description
API get reviews in detail book document
## Reference Requirements
- Product Backlog ID: PBI-39
- Task ID: TI-39.1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
LGTM

File: docs/api-references/post.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## Post Api
+### `GET` Reviews for Book
+```
+/api/books/{id}/reviews
```
Comment (at-huydq) [inline docs/api-references/post.md]: ở dưới get 1 list reviews mà sao đây truyền {id} làm gì e

**Review by ghost — COMMENTED** [review]
check cmt

**Comment by at-huydq** [comment]
LGTM.

**Pull request merged**